src/client/opamArg.mli

```diff
@@ -16,17 +16,64 @@ open Cmdliner
 
 (** {2 Helpers and argument constructors} *)
 
-val mk_flag: ?section:string -> string list -> string -> bool Term.t
+(** {3 CLI versioning} *)
+
+(* Type of the validity of a flag *)
+type validity
+
+val cli2_0: OpamCLIVersion.t
+val cli2_1: OpamCLIVersion.t
+val cli2_2: OpamCLIVersion.t
```

Comment:
```suggestion
```
This appeared for some of the options which were deprecated in opam 2.0, but I don't think it's needed at all?